builder/builder-module.c

```diff
@@ -1298,13 +1318,22 @@ builder_module_build (BuilderModule  *self,
     autotools = TRUE;
   else if (!strcmp (self->buildsystem, "cmake-ninja"))
     cmake_ninja = TRUE;
+  else if (!strcmp (self->buildsystem, "simple"))
+    simple = TRUE;
   else
     {
       g_set_error (error, G_IO_ERROR, G_IO_ERROR_FAILED, "module %s: Invalid buildsystem: \"%s\"",
                    self->name, self->buildsystem);
       return FALSE;
     }
 
+  if (simple && !self->build_commands)
```

Comment:
I think we should just *always* do build-commands inbetween make and make install, rather than make it optional. Most of the time it will not be used, but why not allow it?